pkg/sshutil/sshutil.go

```diff
@@ -90,7 +90,11 @@ func DefaultPubKeys(loadDotSSH bool) ([]PubKey, error) {
 		}
 		entry, err := readPublicKey(f)
 		if err == nil {
-			res = append(res, entry)
+			if strings.ContainsRune(entry.Content, '\n') || !strings.HasPrefix(entry.Content, "ssh-") {
```

Comment:
Containing \n perhaps doesn’t matter, I guess

Reply:
I think so too, but it doesn't hurt, I think.

Leading/trailing whitespace is already stripped, so if there is still a newline in the file, then it has multiple lines with content, which we don't want (I think). E.g. this protects us against somebody doing a `cp ~/.ssh/authorized_keys ~/.ssh/mykeys.pub`, where each line is a valid public key in ssh format.

Anyways, do you want me to remove it? I would prefer to keep it, at least as long as `lima-init` is written in shell and awk. 😄

Reply:
We can keep it. We may change it later if we get a request to do so.